crates/orderbook/openapi.yml

```diff
@@ -308,6 +308,29 @@ paths:
           description: No liquidity was found.
         "500":
           description: Unexpected error.
+  "/api/v1/token/{token}/first_trade_block":
```

Comment:
I think for now I'd prefer to not advertise this endpoint as external people should not rely on it yet IMO.
Also creating individual endpoints for every piece of metadata we might want to compute for a token doesn't seem so nice.
Maybe just have a single endpoint which can be a catch all for different metadata things would be nicer?

Reply:
> Maybe just have a single endpoint which can be a catch all for different metadata things would be nicer?

Yeah, makes sense. Does something like the following look good?
```
GET /api/v1/token/{token}}/metadata?fields=first_trade_block,native_price
```